src/clj2el/cli.clj

```diff
@@ -0,0 +1,29 @@
+(ns clj2el.cli
+  (:require
+   [clj2el.api :refer [clj2el]]
+   [clojure.edn :as edn]
+   [clojure.pprint :refer [pprint]]
+   [clojure.string :as str]))
+
+(defn read-multiple-forms [s]
+  (edn/read-string (str "[" s "]")))
+
+(defn prn-str-multiple-forms [forms]
+  (str/join "\n" (map prn-str forms)))
+
+(defn pprint-str [form] (with-out-str (pprint form)))
+
+(defn pprint-str-multiple-forms [forms]
+  (str/join "\n" (map pprint-str forms)))
+
+(defn -main [& args]
+  (let [argset (set args)
+        multiple? (contains? argset "--multiple") ;; assuming we don't want to depend on babashka.cli
+        pretty? (contains? argset "--pretty")
+        read-fn (if multiple? read-multiple-forms edn/read-string)
```

Comment:
I think it's better to read like it's done in the web-app with edamame: parse-string-all

Reply:
Agreed - manually adding [] to the string to parse didn't feel right.

Reply:
We could also add an api function `clj2el-str` which takes a string and produces a string and then expose this edamame parsing over there instead

Reply:
Using edamame now. Everything just got easier!